packages/zoe/src/zoeService/zoe.js

```diff
@@ -274,7 +274,7 @@ function makeZoe(vatAdminSvc, zcfBundleName = undefined) {
         invitationIssuer,
         zoeInstanceAdminForZcf,
         exportInstanceRecord(),
-        exportIssuerStorage(),
+        getIssuerRecords(),
```

Comment:
I understand why this is a type error. But why wasn't it the equivalent type error without this delta PR?

Reply:
There's two different types, and I think they got conflated in this PR. Here's a fix: https://github.com/Agoric/agoric-sdk/pull/3205